travis: fix failing tests

```
____________________________ test_timezone_selector ____________________________
app = <Flask 'testapp'>
    def test_timezone_selector(app):
        """Test format_datetime."""
        app.config['I18N_LANGUAGES'] = [('da', 'Danish')]
        InvenioI18N(app)
        with app.test_request_context():
            assert format_datetime(datetime(1987, 3, 5, 17, 12)) == \
                'Mar 5, 1987, 5:12:00 PM'
>           assert format_datetime(datetime(1987, 3, 5, 17, 12), 'full') == \
                'Thursday, March 5, 1987 at 5:12:00 PM GMT+00:00'
E           AssertionError: assert 'Thursday, Ma...niversal Time' == 'Thursday, Mar... PM GMT+00:00'
E             - Thursday, March 5, 1987 at 5:12:00 PM Coordinated Universal Time
E             + Thursday, March 5, 1987 at 5:12:00 PM GMT+00:00
tests/test_invenio_i18n.py:89: AssertionError
===================== 1 failed, 44 passed in 11.98 seconds =====================
```

--- a/.gitignore
+++ b/.gitignore
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 
+
 # Idea software family
 .idea/
 
@@ -45,6 +46,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *,cover
+.pytest_cache/
 
 # Translations
 *.mo

--- a/tests/test_invenio_i18n.py
+++ b/tests/test_invenio_i18n.py
@@ -86,8 +86,10 @@ def test_timezone_selector(app):
     with app.test_request_context():
         assert format_datetime(datetime(1987, 3, 5, 17, 12)) == \
             'Mar 5, 1987, 5:12:00 PM'
-        assert format_datetime(datetime(1987, 3, 5, 17, 12), 'full') == \
-            'Thursday, March 5, 1987 at 5:12:00 PM GMT+00:00'
+        # Adds the new date format due to a library update 2
+        assert format_datetime(datetime(1987, 3, 5, 17, 12), 'full') in [
+            'Thursday, March 5, 1987 at 5:12:00 PM GMT+00:00',
+            'Thursday, March 5, 1987 at 5:12:00 PM Coordinated Universal Time']
         assert format_datetime(datetime(1987, 3, 5, 17, 12), 'short') == \
             '3/5/87, 5:12 PM'
         assert format_datetime(datetime(1987, 3, 5, 17, 12), 'dd mm yyy') == \

